### PR TITLE
Check the claim type in pdf & populate the number of claimants field

### PR DESCRIPTION
**Pull request by dan-palmer**
screen shot 2015-03-05 at 15 07 37
screen shot 2015-03-05 at 15 06 53

### CODE REVIEW

File: app/presenters/pdf_form/claim_presenter.rb [hunk 1]
```diff
@@ -41,8 +41,22 @@ def owed?
     claim_types.any? { |type| pay_claims.include? type }
   end
 
+  def type_of_claim
+    if claimant_count > 1
+      'a claimon behalf of more than one person'
```
Comment (stevegraham) [inline app/presenters/pdf_form/claim_presenter.rb]: lél this pdf is crap

**Comment by stevegraham** [comment]
nice work

File: app/presenters/pdf_form/claim_presenter.rb (hunk 1 above)
Comment (dan-palmer) [inline app/presenters/pdf_form/claim_presenter.rb]: hahaha such!

**Comment by dan-palmer** [comment]
Added a query method to claims public interface (shouldn't have forced pushed though, please forgive)

**Pull request merged**